src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -367,9 +370,26 @@ gl_sframe style_transfer::predict(variant_type data,
       case flex_type_enum::VECTOR:
         style_idx = std::move(flex_style_idx.get<flex_vec>());
         break;
+      case flex_type_enum::LIST:
+      {
+        auto list = flex_style_idx.get<flex_list>();
+        style_idx.resize(list.size());
+        std::transform(list.begin(), list.end(),
+                   std::back_inserter(style_idx), [](flexible_type val) {
+                     return static_cast<double>(val.get<flex_float>());
+                   });
```

Comment:
Why do we want to append to `style_idx` after resizing to `list.size()`? The result will be a sequence twice as long as `list`, where the first half of the elements are indeterminate

Reply:
Good catch! Just read up on `back_inserter` didn't know it did that. Would:

```cpp
style_idx.resize(list.size());
std::transform(list.begin(), list.end(), style_idx.begin(), [](flexible_type val) {
   return static_cast<double>(val.get<flex_float>());
});
```

be better than:

```cpp
std::transform(list.begin(), list.end(), std::back_inserter(style_idx), [](flexible_type val) {
   return static_cast<double>(val.get<flex_float>());
});
```

In terms of mallocs and general guidelines?

Reply:
For the latter, you would want to `clear()` and possibly `reserve()` first. The former probably makes more sense in this case, since resizing is cheap for simple types. (Where the default constructor is cheap).